### PR TITLE
postgresql 9.5beta2 (devel)

### PR DESCRIPTION
**Pull request by DocOtak**

### CODE REVIEW

File: Library/Formula/postgresql.rb [hunk 1]
```diff
@@ -1,7 +1,6 @@
 class Postgresql < Formula
   desc "Object-relational database system"
   homepage "https://www.postgresql.org/"
-  revision 2
```
Comment (DomT4) [inline Library/Formula/postgresql.rb]: The revision doesn't get removed on `devel` version updates, only stable ones.

Comment (DomT4) [inline Library/Formula/postgresql.rb]: I know the documentation merged the other day isn't explicit about that; I'll fix that in the morning.

**Comment by DocOtak** [comment]
I've updated the pull to only change the devel source/sha.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/32e08d49dd971d9ea4fa2cf9b1ec62def46323f7. Thank you for your contribution to Homebrew @DocOtak; we appreciate it! :smiley_cat:

**Pull request closed without merging**